### DISCUSSION TITLE
Dropdown button behavior not consistent

### DISCUSSION THREAD

**Discussion by yehwankim23** [community/community#8749] [Other Feature Feedback, Questions, & Ideas] [answered]
When I click on a dropdown button to open the dropdown menu and click on another dropdown button, the already-open dropdown menu doesn't get closed. This behavior is different from the behaviors of the dropdown buttons in the Repositories tab - dropdown menus get closed when I click anywhere outside the menu. I am attaching a screenshot where all three dropdown menus are open. 

![2021-12-13_00h09_28](https://user-images.githubusercontent.com/43813340/145718169-aaf3a00f-5d21-4e15-8115-7f1acc09f6f1.png)

**Comment by brianlovin** [comment, marked as answer]
Thanks for the heads up! We have a tracking issue for this bug, and I'll add your +1 😊 

**Edit:** The fix for this should be live now.
